lib/src/embedded/isolate_dispatcher.dart

```diff
@@ -107,8 +106,12 @@ class IsolateDispatcher {
         _handleError(error, stackTrace);
       },
       onDone: () {
-        _closed = true;
-        _allIsolates.stream.listen((isolate) => isolate.kill());
+        if (_gracefulShutdown) {
+          _closed = true;
+          _allIsolates.stream.listen((isolate) => isolate.kill());
+        } else {
+          exit(exitCode);
+        }
```

Comment:
Since graceful shutdown is disabled on the Dart VM, this is changing the current behavior.

Reply:
"Graceful" here does not mean wait for all inflight request to be completely, - it means wait for child isolates/workers to shutdown first before shutting down main isolate/thread.

In other words, it's only "graceful" for the dart or node vm. This is only required for node, see this earlier comment for explanation: https://github.com/sass/dart-sass/pull/2413#issuecomment-2480235837

From a end user perspective nothing changes, - we have always been "non-graceful".